tools/Ecoregionalization_workflow/indval.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="indval" name="Indicspecies" version="0.1.0+galaxy0" profile="23.0">
+<tool id="indval" name="Indicspecies" version="0.1.0+galaxy0" profile="16.04">
```

Comment:
Why did you decrease it?

Reply:
I tested the tool with planemo locally and with different “profile” and the tool only works with 16.04. I couldn't get it to work with 23.0 or 21.05 (automatically set by planemo tool_init) and I don't really understand why. It seems that Galaxy or the docker container can't find the required packages with these “profile”, but maybe I'm mistaken.

Reply:
mh, this is strange, can you reproduce this locally? If I find time I will check it out.